extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md

```diff
@@ -0,0 +1,59 @@
+# EXT\_mesh\_gpu\_instancing
+
+## Khronos 3D Formats Working Group
```

Comment:
Please change this line to `## Contributors`

Reply:
Done!  Thanks Ed!